.github/workflows/release.yml

```diff
@@ -0,0 +1,62 @@
+#############################################################################
+# GitHub Action to run all phases of the release.
+#
+#############################################################################
+name: "Release"
+on:
+  push:
+    branches:
+      - master
+# TODO: change to manual trigger
+#  workflow_dispatch:
+#    inputs:
+#      version:
+#        description: 'Next version without (-SNAPSHOT)'
+#        required: true
+
+jobs:
```

Comment:
@romani Could you check the order of the jobs and confirm everything is correct?

I am following the diagram:https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/diagrams-manual-trigger.md
and: https://github.com/checkstyle/checkstyle/wiki/Semi-automated-release

Reply:
this https://github.com/checkstyle/checkstyle/wiki/Semi-automated-release is very exact sequence I did last time.

Reply:
For `publish_releasenotes_outside` to work, we need to have already created a GitHub release `checkstyle-10.7.1`. Where do we do that or is it expected to do it manually? https://github.com/stoyanK7/checkstyle/actions/runs/4157241469/jobs/7191609614#step:4:121

Reply:
From what I see, `release.sh` does create it via a call but I do not see an actions for that so I assume I have to create manually?
https://github.com/checkstyle/checkstyle/blob/44d032b713bcae4d6c354f321051a963a466cc8e/release.sh#L68-L74

Reply:
>we need to have already created a GitHub release checkstyle-10.7.1

In release:prepare

Reply:
@romani It does create the tag, but not the GitHub release.

Reply:
I need release like this to be created https://github.com/stoyanK7/checkstyle/releases/tag/checkstyle-10.7.0

but for 10.7.1

I think we need another job for that, no?